src/app/app.config.js

```diff
@@ -0,0 +1,7 @@
+
```

Comment:
Missing [IIFE](https://github.com/johnpapa/angular-styleguide/blob/master/a1/README.md#iife)

Reply:
What's the point of having an IIFE now that we are using webpack ?

Reply:
Yes, no idea. Just we have followed this style for all the explicit dependencies tasks.

Reply:
For the record, here is how webpack transforms the code (before optimizations):

```javascript
"./src/app/app.config.js":
/*!*******************************!*\
  !*** ./src/app/app.config.js ***!
  \*******************************/
/*! no static exports found */
/***/ (function(module, exports, __webpack_require__) {

/* WEBPACK VAR INJECTION */(function(angular) {
  angular.module('esnApp')
    .config(function ($urlRouterProvider) {
      $urlRouterProvider.otherwise(function () {
        return '/calendar';
      });
    });

/* WEBPACK VAR INJECTION */}.call(this, __webpack_require__(/*! ./node_modules/esn-frontend-common-libs/src/angular-common.js */ "./node_modules/esn-frontend-common-libs/src/angular-common.js")))

/***/ }),
```

So:
* we do not need IIFE anymore
* we have to make our styleguide evolve :-)